README.md

```diff
@@ -13,17 +13,20 @@ $ npm install get-media-dimensions
 ## Usage
 
 ```js
-// Several examples of usage.
-// Usually copying and pasting code from the tests and making the code standalone suffices.
-// PLEASE_FILL_IN_HERE
-```
+const getMediaDimensions = require('get-media-dimensions');
 
-## API
+// get video dimensions = { width, height, duration }
+const dimensions = await getMediaDimensions('./video.mp4', 'video');
+const dimensions = await getMediaDimensions('https://somewhere.com/video.mp4', 'video');
 
-PLEASE_FILL_IN_HERE
+// get image dimensions = { width, height }
+const dimensions = await getMediaDimensions('./image.jpg', 'image');
+const dimensions = await getMediaDimensions('https://somewhere.com/image.jpg', 'image');
 
-Note: To regenerate this section from the jsdoc run `npm run docs` and paste
-the output above.
+// get audio dimensions = { duration }
+const dimensions = await getMediaDimensions('./audio.mp3', 'audio');
+const dimensions = await getMediaDimensions('https://somewhere.com/audio.mp3', 'audio');
+```
```

Comment:
I do quite like having the kind of output you get from `jsdoc2markdown` as it has parameter types and, importantly here, output type?